crowdsourcing/viewsets/project.py

```diff
@@ -70,102 +70,184 @@ def list(self, request, *args, **kwargs):
+                                                ) tasks_per_module
+                                                ON m.id=module_id
+                                                LEFT OUTER JOIN (
+                                                    SELECT module_id, COUNT(submitted_tasks) as submitted_tasks
+                                                    FROM (
+                                                        SELECT t.module_id, t.id, submitted_tasks
+                                                        FROM crowdsourcing_task t
+                                                        INNER JOIN ( 
+                                                            SELECT task_id, COUNT(task_id) AS submitted_tasks
+                                                            FROM (
+                                                                SELECT task_worker_id, task_id
+                                                                FROM crowdsourcing_taskworkerresult
+                                                                INNER JOIN (
+                                                                    SELECT task_id, id
+                                                                    FROM crowdsourcing_taskworker
+                                                                    GROUP BY task_id, id
+                                                                ) tw
+                                                                ON tw.id = task_worker_id
+                                                                GROUP BY task_worker_id, task_id
+                                                            ) tw_by_task
+                                                        GROUP BY task_id
+                                                        ) submit_by_task
+                                                        ON t.id=task_id
+                                                    ) submit_with_module
+                                                    GROUP BY module_id
+                                                ) task_count_by_module
+                                                ON task_count_by_module.module_id = m.id
+                                                INNER JOIN (
+                                                    SELECT m.id, (m.repetition * m.task_time * 60 * COUNT(t.id) / num_active_workers) AS module_length
+                                                    FROM crowdsourcing_module m
+                                                    INNER JOIN crowdsourcing_task t ON t.module_id=m.id
+                                                    INNER JOIN (
+                                                        SELECT COUNT(*) AS num_active_workers
+                                                        FROM (
+                                                            SELECT w.id, EXTRACT('EPOCH' FROM NOW() - u.last_login) AS elapsed
```

Comment:
this wont be any help, mainly because last login is does not reflect on when they actually visited our platform, maybe another tracking mechanism could help here